pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -179,36 +180,41 @@
  */
 , alsa-lib
 , bzip2
-, clang
```

Comment:
Nit: could you split the cleanup here into separate commit? This makes the diff clearer and reverts easier.

Reply:
I haven't removed anything. As chayleaf asked for me to sort this I piped it in the sort command. It is just sorted alphabetically.

Reply:
the point is to split the sort in a separate commit before the update, so the update commit diff is cleaner